src/GafferUI/BackdropNodeGadget.cpp

```diff
@@ -276,8 +323,13 @@ void BackdropNodeGadget::renderLayer( Layer layer, const Style *style, RenderRea
 	bound.max /= scale;
 
 	glPushMatrix();
+	glPushAttrib( GL_DEPTH_BUFFER_BIT );
+
+	glEnable( GL_DEPTH_TEST );
+	glDepthFunc( GL_LEQUAL );
 
 	glScalef( scale, scale, scale );
+	glTranslatef( 0, 0, depth( backdrop, bound, ancestor<ViewportGadget>()->getCamera()->getClippingPlanes() ) );
```

Comment:
Not sure if this is a result of this PR, or just existing behaviour now brought to light but we now have trouble with the drawing of backdrop title text when it exceeds the bounds of the backdrop. The Graph Editor background colour is drawn within the bounds of any title text outside of the backdrop bounds, rather than the colour of any other backdrop behind the text. It seems this only occurs when the backdrop in front is drawn before the backdrop behind.

![overlappingBackdropTitleTextIssue](https://github.com/GafferHQ/gaffer/assets/50844517/2ea7bdf4-4ef9-4c9d-8db5-0f3bfa1d173c)

```
import Gaffer
import imath

Gaffer.Metadata.registerValue( parent, "serialiser:milestoneVersion", 1, persistent=False )
Gaffer.Metadata.registerValue( parent, "serialiser:majorVersion", 4, persistent=False )
Gaffer.Metadata.registerValue( parent, "serialiser:minorVersion", 0, persistent=False )
Gaffer.Metadata.registerValue( parent, "serialiser:patchVersion", 0, persistent=False )

__children = {}

__children["Backdrop5"] = Gaffer.Backdrop( "Backdrop5" )
parent.addChild( __children["Backdrop5"] )
__children["Backdrop5"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
__children["Backdrop5"].addChild( Gaffer.Box2fPlug( "__uiBound", defaultValue = imath.Box2f( imath.V2f( -10, -10 ), imath.V2f( 10, 10 ) ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
__children["Backdrop6"] = Gaffer.Backdrop( "Backdrop6" )
parent.addChild( __children["Backdrop6"] )
__children["Backdrop6"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
__children["Backdrop6"].addChild( Gaffer.Box2fPlug( "__uiBound", defaultValue = imath.Box2f( imath.V2f( -10, -10 ), imath.V2f( 10, 10 ) ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
__children["Backdrop7"] = Gaffer.Backdrop( "Backdrop7" )
parent.addChild( __children["Backdrop7"] )
__children["Backdrop7"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
__children["Backdrop7"].addChild( Gaffer.Box2fPlug( "__uiBound", defaultValue = imath.Box2f( imath.V2f( -10, -10 ), imath.V2f( 10, 10 ) ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
Gaffer.Metadata.registerValue( __children["Backdrop5"], 'nodeGadget:color', imath.Color3f( 0.479999989, 0.245000005, 0.245000005 ) )
__children["Backdrop5"]["title"].setValue( 'An overly long and overlapping backdrop title' )
__children["Backdrop5"]["__uiPosition"].setValue( imath.V2f( 71.2871857, -20.0904503 ) )
__children["Backdrop5"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -34.4277039, -12.1572971 ), imath.V2f( 2.22060966, 16.3704109 ) ) )
Gaffer.Metadata.registerValue( __children["Backdrop6"], 'nodeGadget:color', imath.Color3f( 0.0922499895, 0.224999994, 0.0935774967 ) )
__children["Backdrop6"]["description"].setValue( '\n' )
__children["Backdrop6"]["__uiPosition"].setValue( imath.V2f( 52.8053017, -16.673008 ) )
__children["Backdrop6"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -25.4570084, -22.1343002 ), imath.V2f( 84.926178, 18.0646038 ) ) )
Gaffer.Metadata.registerValue( __children["Backdrop7"], 'nodeGadget:color', imath.Color3f( 0.479999989, 0.245000005, 0.245000005 ) )
__children["Backdrop7"]["title"].setValue( 'An overly long and overlapping backdrop title' )
__children["Backdrop7"]["__uiPosition"].setValue( imath.V2f( 127.198906, -19.7091179 ) )
__children["Backdrop7"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -34.4277039, -12.1572971 ), imath.V2f( 2.22060966, 16.3704109 ) ) )


del __children
```